### ISSUE TITLE
Error deploying starter intranet solution with May intermediate 1 release of PnP PowerShell: Invalid list template for carousel list

### ISSUE THREAD

**Issue by RGrizzzz** [SharePoint/PnP#1787]
Error deploying Starter Intranet solution with May Intermediate release 1 of PnP Powershell. I was getting an exception on deploying/applying the list template for the carousel list in the provisioning (engine) steps of the root web. I switched back to the April CU, and it works fine. 

Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. If you have a actual question, we would ask you to use [SharePoint Developer Group](http://aka.ms/sppnp-community) at Microsoft Tech Community. Thanks!

####Which PnP repository should you use to report the issue?####
*Please navigate to the appropriate repository by clicking on the link below and create your issue there. You can delete this section after you have navigated to the correct repository.*

- PnP Samples - https://github.com/OfficeDev/PnP
- PnP Sites Core - https://github.com/OfficeDev/PnP-Sites-Core
- PnP JS Core - https://github.com/OfficeDev/PnP-JS-Core
- PnP PowerShell - https://github.com/OfficeDev/PnP-PowerShell

#### Category
[ x] Bug
[ ] Enhancement

#### Environment
[ x] Office 365 / SharePoint Online
[ ] SharePoint 2016
[ ] SharePoint 2013

If SharePoint on-premises, what's exact CU version: 

#### Expected or Desired Behavior
_If you are reporting a bug, please describe the expected behavior. If you are suggesting an enhancement please
describe thoroughly the enhancement, how it can be achieved, and expected benefit._

#### Observed Behavior
_If you are reporting a bug, please describe the behavior you expected to occur when performing the action. If you are making a suggestion, you can delete this section._

#### Steps to Reproduce
_If you are reporting a bug please describe the steps to reproduce the bug in sufficient detail to allow testing. Only way to fix things properly, is to have sufficient details to reproduce it. If you are making a suggestion, you can delete this section._

#### Submission Guidelines
_Delete this section after reading_
- All suggestions or bugs are welcome, please let us know what's on your mind.
- If you are reporting any issues around PnP Core Component, please reproduce the issue with latest release.
- If you are reporting issue around PnP Provisioning Engine, please share the xml template, if possible.
- If you are reporting an issue around any of the samples, please ensure that you have clear reference on the sample and possibly code file, which should be fixed.
- If you have a general question, please use the [SharePoint Developer Group](http://aka.ms/sppnp-community) at Microsoft Tech Community.
- Remember to include sufficient details and context.
- If you have multiple suggestions or bugs please submit them in seperate bugs so we can track resolution.

Thanks for your contribution! Sharing is caring.

**Comment by FranckyC** [comment]
Hi @RGrizzzz, thanks to flag this issue. Is there a reason you need the latest version of the PnP PowerShell?
Generally, the latest version is not really necessary to deploy the solution since it does not use the new features anyway. Latest versions are more targeted for the "modern" SharePoint in Office 365. Like you said, using an "old" version should to the trick. I suggest you to post this issue directly in the https://github.com/SharePoint/PnP-Sites-Core repository since it is not related to the PnP Starter intranet. Thanks!

**Comment by RGrizzzz** [comment]
With April 2018, ran into this for both English and French. When English failed, I kicked off the script again, and it started creating the French version. French failed too. Started a 3rd time, and everything completed. The failures in those methods don't look serious. The site seems to be rendering.

Creating and configuring the sub web for the language 'en'...
Configuring the pages library...Configure-Web : Cannot process argument transformation on parameter 'Web'. Cannot convert the "Microsoft.SharePoint.Client.Web" value of type "Microsoft.SharePoint.Client.Web" to type "Microsoft.SharePoint.Client.Web".
At C:\Users\rgriz\Desktop\POC Demo\PnP-master\Solutions\Business.StarterIntranet\Setup-Web.ps1:332 char:13
+             Configure-Web -LanguageInfo $CurrentLanguage
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Configure-Web], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Configure-Web

**Comment by FranckyC** [comment]
@RGrizzzz good point. The README file doesn't mention this trick. However, since I'm not the one who implement the PnP PowerShell module, I can't guarantee every release will be compatible with the solution ;). I seems your error is related to a conflict between the SharePoint Online PowerShell SDK and PnP. I suggest you to run the installation without the SPO module loaded.

To deploy the solution, currently I use the Feb 2018 version. https://github.com/SharePoint/PnP-PowerShell/releases/tag/2.23.1802.0
